doc/wolfssl/server.pem

```diff
@@ -0,0 +1,81 @@
+-----BEGIN RSA PRIVATE KEY-----
```

Comment:
All over this PR, please do not include private keys.  People may copy configs and copy keys, even if these were meant only as an example.

Please instead provide instructions how to generate the certificate files.